lib/sqlalchemy/sql/ddl.py

```diff
@@ -470,14 +467,12 @@ def __init__(
         self,
         name,
         quote=None,
-        if_exists=False,
```

Comment:
**mike bayer** (zzzeek) wrote:

this was weird, I have no idea why this was here.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4067